src/Identifier/TokenIdentifier.php

```diff
@@ -12,34 +12,32 @@
  */
 namespace Authentication\Identifier;
 
-use Cake\ORM\TableRegistry;
-use InvalidArgumentException;
-use RuntimeException;
+use Authentication\Identifier\Backend\DatasourceAwareTrait;
 
 /**
  * Token Identifier
  */
 class TokenIdentifier extends AbstractIdentifier
 {
 
+    use DatasourceAwareTrait;
+
     /**
-     * Default configuration
+     * Default configuration.
      *
      * @var array
      */
     protected $_defaultConfig = [
         'tokenField' => 'token',
         'dataField' => 'token',
-        'userModel' => 'Users',
-        'finder' => 'all',
-        'tokenVerification' => 'Orm'
+        'datasource' => 'Authentication.Orm'
     ];
 
     /**
-     * Identify
+     * Identify method.
      *
-     * @param array $data Authentication credentials
-     * @return \Cake\Datasource\EntityInterface|null
+     * @param array $data Authentication credentials.
+     * @return \ArrayAccess|null
```

Comment:
Any reason you've already changed this without a decision on if we use \ArrayAccess or \EntityInterface?